packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/CreatePage/index.js

```diff
@@ -85,10 +82,7 @@ const CreatePage = () => {
         }
 
         if (res.data.id && !isEmpty(permissionsToSend)) {
-          await request(`/admin/roles/${res.data.id}/permissions`, {
-            method: 'PUT',
-            body: { permissions: permissionsToSend },
-          });
+          await put(`/admin/roles/${res.data.id}/permissions`, { permissions: permissionsToSend });
```

Comment:
I think this needs to be wrapped in try / catch, because axios would throw where request does not.